tempesta_fw/sched/tfw_sched_hash.c

```diff
@@ -197,7 +197,7 @@ tfw_sched_hash_update_data(TfwSrvGroup *sg)
 		}
 	}
 
-	BUG_ON(hash_idx >= TFW_SG_MAX_CONN);
+	BUG_ON(hash_idx > TFW_SG_MAX_CONN + 1);
```

Comment:
Yeah, but still should be ">=" since we allocate n + 1 elements in TFW_CONN_HASH_LIST_SIZE